fastlane/Fastfile

```diff
@@ -6,25 +6,112 @@
+# to "status-adhoc" output folder
+def build_ios_adhoc
+    match(
+        type: "adhoc",
+        force_for_new_devices: true,
+        readonly: true,
+        keychain_name: "login.keychain"
+    )
+    build_ios_app(
+        scheme: "StatusIm",
+        workspace: "ios/StatusIm.xcworkspace",
+        configuration: "Release",
+        clean: true,
+        export_method: "ad-hoc",
+        output_directory: "status-adhoc"
+    )
+end
 
 
 platform :ios do
-  desc "Deploy a new internal build to iTunes Connect"
+  desc "`fastlane ios adhoc` - ad-hoc lane for iOS."
+  desc "This lane is used PRs, Releases, etc."
+  desc "It creates an .ipa that can be used by a list of devices, registeded in the App Store Connect."
+  desc "This .ipa is ready to be distibuted through diawi.com"
+  lane :adhoc do
+    unlock_keychain_if_needed
+    build_ios_adhoc
+  end
+
+  desc "`fastlane ios nightly` - makes a new nightly in TestFlight"
+  desc "This lane builds a new nightly and uploads it to TestFlight"
+  desc "It also leaves an .ipa that is ad-hoc signed (can be uploaded to diawi)"
   lane :nightly do
+    unlock_keychain_if_needed
+
+    match(
+        type: "appstore",
+        force_for_new_devices: true,
+        readonly: true,
+        keychain_name: "login.keychain"
+    )
+    build_ios_app(
+        scheme: "StatusIm",
+        workspace: "ios/StatusIm.xcworkspace",
+        configuration: "Release",
+        clean: true,
+        export_method: "app-store",
+        output_directory: "status_appstore"
+    )
     upload_to_testflight(
       ipa: "status_appstore/StatusIm.ipa"
     )
-    slack(message: "New nightly build uploaded to iTunes Connect")
+    slack(message: "New nightly build uploaded to TestFlight")
+
+    # additional .ipa is for diawi
+    # we have to re-build it because it uses different config
+    build_ios_adhoc
   end
+
+  desc "`fastlane ios release` builds a release & uploads it to TestFlight"
   lane :release do
+    match(
+        type: "appstore",
+        force_for_new_devices: true,
+        readonly: true,
+        keychain_name: "login.keychain"
+    )
+    build_ios_app(
+        scheme: "StatusIm",
+        workspace: "ios/StatusIm.xcworkspace",
+        configuration: "Release",
+        clean: true,
+        export_method: "app-store",
+        output_directory: "status_appstore",
+        export_options: {
+            "combileBitcode": true,
+            "uploadBitcode": false,
+            "ITSAppUsesNonExemptEncryption": false
+        }
+    )
     upload_to_testflight(
       ipa: "status_appstore/StatusIm.ipa"
     )
-    slack(message: "New release build uploaded to iTunes Connect")
+    slack(message: "New release build uploaded to TestFlight")
   end
-  desc "Remove inactive TestFlight users"
+
+  desc "`fastlane ios clean` - remove inactive TestFlight users"
+  desc "uses custom plugin, installed via"
+  desc "`sudo get install fastlane-plugin-clean_testflight_testers`"
```

Comment:
Plugin [install docs](https://github.com/KrauseFx/fastlane-plugin-clean_testflight_testers/blob/master/README.md) mentions `fastlane add_plugin clean_testflight_testers`

Reply:
it is the same but probably a better syntax
feel free to fix it an submit a PR :-P

Reply:
I tried `sudo get install fastlane-plugin-clean_testflight_testers` but got `command not found` :(